homeassistant/generated/bluetooth.py

```diff
@@ -118,6 +118,16 @@
         "local_name": "tps",
         "connectable": False
     },
+    {
+        "domain": "melnor",
+        "local_name": "YM*"
+    },
+    {
+        "domain": "melnor",
+        "manufacturer_data_start": [
```

Comment:
You'll need to set a `manufacturer_id` as well otherwise this matcher will be too broad